### DISCUSSION TITLE
KWB Goslar POIS for Wildemann does not deliver meaningful data

### DISCUSSION THREAD

**Discussion by StefanBu1970** [mampfes/hacs_waste_collection_schedule#2057] [Q&A]
I am trying to integrate the waste calendar form KWB Goslar for the Location Wildemann.
POIS Value from KWB Website is 2523.700, but this seems to deliver multiple results.

If I use other towns nearby, the values look good.

Any Idea, if the POIS value is wrong? Where can I verify the POIS value?

![grafik](https://github.com/mampfes/hacs_waste_collection_schedule/assets/126359249/349d02e5-7a28-472c-b164-9d0827a370d2)

**Comment by 5ila5** [comment]
The source downloads from the url like seen below. You can check the parameters if the ICAL file download url has the same parameters by either coping the download URL of the file (Firefox: Right-click in the downloads drop Down -> copy download link) or by inspecting the network traffic while downloading the ICS file (Developer tools (F12 or right click inspect) in the network section)

```python
    requests.get(
            url="https://www.kwb-goslar.de/output/options.php",
            params={
                "ModID": "48",
                "call": "ical",
                "pois": self.pois,
            }
    )
```

which results in a GET request to `https://www.kwb-goslar.de/output/options.php?ModID=48&call=ical&pois={POIS}`
